tests/system/providers/google/cloud/gcs/example_sftp_to_gcs.py

```diff
@@ -22,39 +22,61 @@
 
 import os
 from datetime import datetime
+from pathlib import Path
 
 from airflow import models
+from airflow.models.baseoperator import chain
+from airflow.operators.bash import BashOperator
+from airflow.providers.google.cloud.operators.gcs import GCSCreateBucketOperator, GCSDeleteBucketOperator
 from airflow.providers.google.cloud.transfers.sftp_to_gcs import SFTPToGCSOperator
+from airflow.utils.trigger_rule import TriggerRule
 
-BUCKET_SRC = os.environ.get("GCP_GCS_BUCKET_1_SRC", "test-sftp-gcs")
+ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID")
+PROJECT_ID = os.environ.get("SYSTEM_TESTS_GCP_PROJECT")
 
-TMP_PATH = "/tmp"
+DAG_ID = "example_sftp_to_gcs"
+BUCKET_NAME = f"bucket-{DAG_ID}-{ENV_ID}"
+
+TMP_PATH = "tmp"
 DIR = "tests_sftp_hook_dir"
 SUBDIR = "subdir"
 
 OBJECT_SRC_1 = "parent-1.bin"
 OBJECT_SRC_2 = "parent-2.bin"
-OBJECT_SRC_3 = "parent-3.txt"
+
+CURRENT_FOLDER = Path(__file__).parent
+LOCAL_PATH = str(Path(CURRENT_FOLDER) / "resources")
+
+FILE_LOCAL_PATH = str(Path(LOCAL_PATH) / TMP_PATH / DIR)
+FILE_NAME = "tmp.tar.gz"
```

Comment:
Do we need to use zipped file for testing transfering the data between SFTP and Google storage? Or maybe it could be an empty txt file (and then we can also remove BashOperator that unzip the file).

Reply:
tmp.tar.gz is needed because we need more files for testing